templates/style/base.scss

```diff
@@ -444,87 +444,118 @@ div.cratesfyi-package-container {
     border-bottom: 1px solid $color-border;
     margin-bottom: 20px;
 
-    h1 {
-        margin: 0;
-        padding: 15px 0 0 14px;
+    .container {
+        display: flex;
+        align-items: center;
 
-        &.no-description {
-            padding-bottom: 15px;
-        }
-    }
+        .description-container {
+            flex-grow: 3;
```

Comment:
Without this it looks a lot better IMO, the 'Documentation' tab is centered:
![image](https://user-images.githubusercontent.com/23638587/92479868-1d85c480-f1b2-11ea-83f6-8724ebdce776.png)

Reply:
I'm pretty sure this way the button will jump around the page if the description is longer.

Reply:
An example of a longer description: <https://docs.rs/crate/tokio/0.2.22>

Reply:
This is what it looks like on mobile: 
![image](https://user-images.githubusercontent.com/23638587/92480208-96851c00-f1b2-11ea-9c9c-01283a816961.png)

Reply:
> I'm pretty sure this way the button will jump around the page if the description is longer.

Hmm, maybe we can limit the length of the description and show this button after?

Reply:
It's kind of the case. I just don't like the fact that the button position might change, which is why I added `flex-grow: 3` so like that, whatever the content length, it remains on the right.

Reply:
Ok yeah, with `width: 50%;` it looks consistent:
![image](https://user-images.githubusercontent.com/23638587/92481322-2f686700-f1b4-11ea-8686-702f3082ee3d.png)
![image](https://user-images.githubusercontent.com/23638587/92481327-342d1b00-f1b4-11ea-8467-17b56e20f124.png)

Reply:
I actually prefer it with the text limited as well, it makes the lines easier to read when they're shorter.

Reply:
Fine by me. We'll have both then! :)

Reply:
It feels... weird. Dunno why, but I'm not sure it's the right approach :/

Reply:
I don't like it either. Didn't add the line yet so feel free to debate about it. :)

Reply:
We decided to instead keep the right position, but make the button more noticable by adding a black background.